docs/contributor/cli.md

```
@@ -780,7 +780,7 @@ flowchart TD
 
 #### Bundler Architecture
 
-**BaseBundler Helper Pattern:**
+##### BaseBundler Helper Pattern
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)